about.html

```diff
@@ -22,25 +52,23 @@
           <li class="nav-menu-item">
             <a href="index.html">HOME</a>
           </li>
-
           <li class="nav-menu-item">
             <a href="./about.html">ABOUT</a>
           </li>
-
-          <li class="nav-menu-item">
-            <a href="./index.html#roadmap-block">ROADMAP</a>
-          </li>
-
+          <li class="nav-menu-item"><a href="#roadmap-block">ROADMAP</a></li>
           <li class="nav-menu-item">
             <a href="./catalog.html">COLLECTION</a>
           </li>
-
           <li class="nav-menu-item">
             <a href="./faqs.html">FAQS</a>
           </li>
-
           <li class="nav-menu-item">
-            <a href="#" class="dropdown">PAGES</a>
+            <a href="#" class="dropdown">
+              PAGES
+              <svg class="icon arrow-dwn">
```

Comment:
I think no need to reduce class name - `arrow-down` more clear

Reply:
Thank you for comments. I will try to pay more attention on class names. 

#fixed